coalib/settings/SectionFilling.py

```diff
@@ -71,6 +71,10 @@ def fill_section(section, acquire_settings, log_printer, bears):
     prel_needed_settings = {}
     for bear in bears:
         needed = bear.get_non_optional_settings()
+        # FIXME: See https://github.com/coala/coala/issues/3149
+        for bear_dep in bear.BEAR_DEPS:
+            needed.update(bear_dep.get_non_optional_settings())
```

Comment:
Right.  This is a temporary fix, and should be unnecessary if https://github.com/coala/coala-bears/pull/1100 is an acceptable approach.

https://github.com/coala/coala/issues/3149 needs design.